README.md

```diff
@@ -50,13 +50,24 @@ Only one instance of the plugin is *possible*.
 - Enabled
   - bool
   - default: false
-  - enable the plugin
+  - Enable the plugin
 - LogLevel
   - string
   - default: `INFO`, expected values are: `INFO`, `DEBUG`
 - CrowdsecMode
   - string
   - default: `live`, expected values are: `none`, `live`, `stream`, `alone`
+- CrowdsecAppsecEnabled
+  - bool
+  - default: false
+  - Enable Crowdsec Appsec Server (WAF) 
+- CrowdsecAppsecScheme
```

Comment:
We miss documentation for Appsec module, i'll add some content in the read me and links to crowdsec documentation

Reply:
Done